packages/e2e-tests/specs/editor/various/sidebar.test.js

```diff
@@ -1,171 +0,0 @@
-/**
- * WordPress dependencies
- */
-import {
-	clearLocalStorage,
-	createNewPost,
-	findSidebarPanelWithTitle,
-	enableFocusLossObservation,
-	disableFocusLossObservation,
-	openDocumentSettingsSidebar,
-	pressKeyWithModifier,
-	setBrowserViewport,
-} from '@wordpress/e2e-test-utils';
-
-const SIDEBAR_SELECTOR = '.edit-post-sidebar';
-const ACTIVE_SIDEBAR_TAB_SELECTOR =
-	'div[aria-label="Editor settings"] [role="tab"][aria-selected="true"]';
-const ACTIVE_SIDEBAR_BUTTON_TEXT = 'Post';
-
-describe( 'Sidebar', () => {
-	afterEach( () => {
-		disableFocusLossObservation();
-	} );
-
-	it( 'should have sidebar visible at the start with document sidebar active on desktop', async () => {
-		await setBrowserViewport( 'large' );
-		await clearLocalStorage();
-		await createNewPost();
-		await enableFocusLossObservation();
```

Comment:
The focus loss observer was introduced in #10917. This was the only test where this method was used, but it's unclear what it was testing.

I've omitted it from the migration for now. If we want a focus loss test, we can write a more explicit one with Playwright.